tests/acceptance/DeleteNewsletterCest.php

```diff
@@ -52,30 +66,62 @@ public function deleteFormPermanently(\AcceptanceTester $i) {
     $i->clickItemRowActionByItemName($newsletterName, 'Delete Permanently');
     $i->waitForText('1 email was permanently deleted.');
     $i->waitForElementNotVisible($newsletterName);
-
     $i->waitForText($newsletterName . '2');
+    $i->waitForText($newsletterName . '3');
+    // click to select all newsletters
+    $i->click('.mailpoet-form-checkbox-control');
+    $i->click('Delete Permanently');
+    $i->waitForText('2 emails were permanently deleted.');
+    $i->waitForElement('[data-automation-id="filters_all"]');
+    $i->waitForText($newsletterName . '4');
   }
 
   public function emptyTrash(\AcceptanceTester $i) {
     $newsletterName = 'Goodbye Forever Newsletter';
     $i->wantTo('Empty a trash on Newsletters page');
     $newsletter = new Newsletter();
     $newsletter->withSubject($newsletterName)->withDeleted()->create();
+    $newsletter->withSubject($newsletterName . '2')->withDeleted()->create();
     $newsletter = new Newsletter();
-    $newsletter->withSubject($newsletterName . '2')->create();
-
+    $newsletter->withSubject($newsletterName . '3')->create();
     $i->login();
     $i->amOnMailpoetPage('Emails');
     $i->waitForElement('[data-automation-id="filters_trash"]');
     $i->click('[data-automation-id="filters_trash"]');
     $i->waitForText($newsletterName);
-
     $i->click('[data-automation-id="empty_trash"]');
-
-    $i->waitForText('1 email was permanently deleted.');
+    $i->waitForText('2 emails were permanently deleted.');
     $i->waitForElementNotVisible($newsletterName);
     $i->click('[data-automation-id="filters_all"]');
+    $i->waitForText($newsletterName . '3');
+  }
 
-    $i->waitForText($newsletterName . '2');
+  public function selectAllAvailableNewslettersAndDelete(\AcceptanceTester $i) {
+    $newsletterName = 'Sample Newsletter';
+    $newsletter = new Newsletter();
+    for ($itemCount = 1; $itemCount <= 22; $itemCount++) {
+      $newsletter->withSubject($newsletterName . $itemCount)->withSentStatus()->create();
+    }
+    $i->wantTo('Select all available newsletters and proceed with deletion');
```

Comment:
Usually `$i->wantTo()` call is on the first line of the test case, just for better readability.